server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts

```diff
@@ -299,89 +295,167 @@ export class AlfredResourcesFactory implements core.IResourcesFactory<AlfredReso
 		);
 		const restCreateDocThrottler = configureThrottler(restApiCreateDocThrottleConfig);
-
 		const restApiGetDeltasThrottleConfig = utils.getThrottleConfig(
-			config.get("alfred:throttling:restCallsPerCluster:getDeltas"),
+			config.get("alfred:throttling:perCluster:getDeltas"),
 		);
 		const restGetDeltasThrottler = configureThrottler(restApiGetDeltasThrottleConfig);
-
 		const restApiGetSessionThrottleConfig = utils.getThrottleConfig(
-			config.get("alfred:throttling:restCallsPerCluster:getSession"),
+			config.get("alfred:throttling:perCluster:getSession"),
 		);
 		const restGetSessionThrottler = configureThrottler(restApiGetSessionThrottleConfig);
-
-		const restClusterThrottlers = new Map<string, core.IThrottler>();
-		restClusterThrottlers.set(Constants.createDocThrottleIdPrefix, restCreateDocThrottler);
-		restClusterThrottlers.set(Constants.getDeltasThrottleIdPrefix, restGetDeltasThrottler);
-		restClusterThrottlers.set(Constants.getSessionThrottleIdPrefix, restGetSessionThrottler);
-
-		// Socket Connection Throttler
-		const socketConnectionThrottleConfigPerTenant = utils.getThrottleConfig(
-			config.get("alfred:throttling:socketConnectionsPerTenant"),
-		);
-		const socketConnectTenantThrottler = configureThrottler(
-			socketConnectionThrottleConfigPerTenant,
-		);
-
 		const socketConnectionThrottleConfigPerCluster = utils.getThrottleConfig(
-			config.get("alfred:throttling:socketConnectionsPerCluster"),
+			config.get("alfred:throttling:perCluster:socketConnections"),
 		);
 		const socketConnectClusterThrottler = configureThrottler(
 			socketConnectionThrottleConfigPerCluster,
 		);
-
-		// Socket SubmitOp Throttler
 		const submitOpThrottleConfig = utils.getThrottleConfig(
-			config.get("alfred:throttling:submitOps"),
+			config.get("alfred:throttling:perCluster:submitOps"),
 		);
 		const socketSubmitOpThrottler = configureThrottler(submitOpThrottleConfig);
-
-		// Socket SubmitSignal Throttler
 		const submitSignalThrottleConfig = utils.getThrottleConfig(
-			config.get("alfred:throttling:submitSignals"),
+			config.get("alfred:throttling:perCluster:submitSignals"),
 		);
 		const socketSubmitSignalThrottler = configureThrottler(submitSignalThrottleConfig);
+
+		const clusterThrottlers = new Map<string, core.IThrottler>();
+		clusterThrottlers.set(Constants.createDocThrottleIdPrefix, restCreateDocThrottler);
+		clusterThrottlers.set(Constants.getDeltasThrottleIdPrefix, restGetDeltasThrottler);
+		clusterThrottlers.set(Constants.getSessionThrottleIdPrefix, restGetSessionThrottler);
+		clusterThrottlers.set(
+			Constants.socketConnectionsThrottleIdPrefix,
+			socketConnectClusterThrottler,
+		);
+		clusterThrottlers.set(Constants.submitOpsThrottleIdPrefix, socketSubmitOpThrottler);
+		clusterThrottlers.set(Constants.submitSignalThrottleIdPrefix, socketSubmitSignalThrottler);
+
+		// Tenant Group 1 Rest API Throttlers
+		const restApiCreateDocThrottleConfigGroup1 = utils.getThrottleConfig(
```

Comment:
(*non-blocking*)
I think we can remove rest prefix and add tenant to the name instead

> CreateDocThrottleConfigTenantGroup1

Reply:
Done. :)